components/com_users/tmpl/login/default_login.php

```diff
@@ -59,15 +59,13 @@
 			<?php endif; ?>
 
 			<?php if (PluginHelper::isEnabled('system', 'remember')) : ?>
-				<div  class="com-users-login__remember control-group">
-					<div class="control-label">
-						<label for="remember">
+				<div  class="com-users-login__remember">					
```

Comment:
```suggestion
				<div class="com-users-login__remember">					
```

please remove extra space

Reply:
> 
> 
> please remove extra space

was not seen before (under Issue)